pkg/cloudprovider/provider/digitalocean/provider.go

```diff
@@ -318,6 +318,11 @@ func (d *doInstance) ID() string {
 	return strconv.Itoa(d.droplet.ID)
 }
 
+func (d *doInstance) PublicAddress() string {
+	//TODO: Implement this
+	return "Not implemented"
```

Comment:
As long as we don't return an error here, can you do a 

`panic("not implemented")`

just in case?

Reply:
Hm, don't you think killing off the whole controller just because of that would be a bit over the top? Maybe it would be smarter to return an empty string, thought.